osi_surfaceline.proto

```diff
@@ -0,0 +1,169 @@
+syntax = "proto2";
+
+option optimize_for = SPEED;
+
+import "osi_common.proto";
+import "osi_lane.proto";
+
+package osi3;
+
+// A lane's surface line (as a list of segments).
```

Comment:
Failing build test: \brief section required for messages with longer description.

Reply:
It seems that \brief is mandatory for any message